crates/iroha_config/Cargo.toml

```diff
@@ -40,3 +40,4 @@ stderrlog = "0.6.0"
 [dev-dependencies]
 expect-test = { workspace = true }
 assertables = { workspace = true }
+iroha_crypto = { workspace = true, features = ["rand"] }
```

Comment:
This doesn't seem to me to be used anywhere

Reply:
It is used in config tests for random keypairs.